### PR TITLE
Adding documentation for Qualcomm Robotics RB5 development kit

### PR DESCRIPTION
**Pull request by rbmistry**
Adding documentation for Qualcomm Robotics RB5 development kit

### CODE REVIEW

File: consumer/dragonboard/README.md [hunk 1]
```diff
@@ -1,23 +1 @@
...
| <img src="https://www.96boards.org/product/ce/rb3-platform/images/db845c-front-sd.png?raw=true" data-canonical-src="https://www.96boards.org/product/ce/rb3-platform/images/db845c-front-sd.png?raw=true" width="200" height="130" /><br> **Qualcomm RB3** | Board based on Qualcomm® Snapdragon™ 845 processor  | [Documentation](dragonboard/dragonboard845c/)<br> |
| <img src="https://github.com/96boards/documentation/blob/master/consumer/dragonboard/dragonboard820c/additional-docs/images/images-board/sd/dragonboard820c-front-sd.png?raw=true" data-canonical-src="https://github.com/96boards/documentation/blob/master/consumer/dragonboard/dragonboard820c/additional-docs/images/images-board/sd/dragonboard820c-front-sd.png?raw=true" width="220" height="200" /><br> **DragonBoard 820c** | Board based on Qualcomm® APQ8016E processor  | [Documentation](dragonboard820c/)<br> |
| <img src="https://i.imgur.com/4a5GXRd.png" data-canonical-src="https://i.imgur.com/4a5GXRd.png" width="200" height="130" /><br> **DragonBoard 410c** | Board based on Qualcomm® Snapdragon™ 410E processor  | [Documentation](dragonboard410c/)<br>|

</div>
```
Comment (ric96) [inline consumer/dragonboard/README.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
...

- [Downloads from Thundercomm](https://www.thundercomm.com/app_en/product/1590131656070623)
   - Download the SDK Manager tool from Thundercomm's website to build and flash the images on the development board.
- [Software Tools and Drivers from Qualcomm](https://www.qualcomm.com/products/qualcomm-robotics-rb5-platform/software)
   - Tools and software to support the development platform.


## Support
- [Community Forums](https://developer.qualcomm.com/forums/hardware/robotics/qualcomm-robotics-rb5-kit)
   - From bug reports and current issues, to forum access and other useful resources, we want to help you find answers
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/README.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/downloads/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
...
### Third Party Operating Systems

Other operating systems supported on Qualcomm Robotics RB5 development kit, offered by partners.

- [LU OS from Thundercomm](https://www.thundercomm.com/app_en/product/1590131656070623)
   - Download the SDK Manager tool from Thundercomm's website to build and flash the LU images on the development board.

Continue to [Installation Page](../installation/)

***
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/downloads/README.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/downloads/open-embedded.md [hunk 4]
```diff
@@ -0,0 +1 @@
...

| RPB-Wayland                                                                                                                             |
|:----------------------------------------------------------------------------------------------------------------------------------------|
| [Boot Image](https://releases.linaro.org/96boards/rb5/linaro/openembedded/latest/rpb-wayland/boot--*-r0-qrb5165-rb5-*-*.img)                                                                                                               |
| [Desktop](https://releases.linaro.org/96boards/rb5/linaro/openembedded/latest/rpb-wayland/rpb-weston-image-qrb5165-rb5-*-*.rootfs.ext4.gz) or [Console](https://releases.linaro.org/96boards/rb5/linaro/openembedded/latest/rpb-wayland/rpb-console-image-qrb5165-rb5-*-*.rootfs.ext4.gz)                                                       |
| [Build Folder](https://releases.linaro.org/96boards/rb5/linaro/openembedded/latest/rpb-wayland/)                 |

> NOTE: Only download one root file system (Console or Desktop). You should match the type of rootfs to the boot image you downloaded above.

#### Continue to [Installation page](../installation/)
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/downloads/open-embedded.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/installation/README.md [hunk 5]
```diff
@@ -0,0 +1 @@
...
### Board Recovery

Attempt to recover a board from a bad flash resulting in a soft-bricked board.
- [Board Recovery](board-recovery.md)

***

### Third Party OS

- [LE OS Installation](le.md)
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/installation/README.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/installation/le.md [hunk 6]
```diff
@@ -0,0 +1 @@
...
fastboot flash userdata sda845-usrfs_1.ext4
fastboot flash persist sda845-persist_1.ext4
```

#### **Step 5**: Reboot the Qualcomm Robotics RB5 development kit

- ```fastboot reboot```

**Congratulations! You are now booting your newly installed OS directly
from UFS on the Qualcomm Robotics RB5 development kit!**
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/installation/le.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

File: consumer/dragonboard/qualcomm-robotics-rb5/installation/linux-fastboot.md [hunk 7]
```diff
@@ -0,0 +1 @@
...
- Unplug power to RB5
- Ensure HDMI connection to monitor
- Ensure keyboard and/or mouse connection (Depending on your rootfs selection)
- Plug power back into RB5
- Wait for board to boot up
- Board will boot into either command line or desktop depending on rootfs

**Note:** For Linaro/Debian the **username** and **password** are both **“linaro”** when the login information is requested.

**Congratulations! You are now booting your newly installed OS directly from UFS on the RB5!**
```
Comment (ric96) [inline consumer/dragonboard/qualcomm-robotics-rb5/installation/linux-fastboot.md:1]: formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`

**Review by ric96 — CHANGES_REQUESTED** [review]
just the odd `^M` characters, rest seem fine.

**Comment by ric96** [comment]
@rbmistry there are multiple files with the formatting issue.

File: consumer/dragonboard/README.md (hunk 1 above)
Comment (rbmistry) [reply consumer/dragonboard/README.md]: Hi Sahaj,

I had originally used Windows Desktop to commit the changes. I have a feeling the characters got added automatically somehow. If I open the files in the browser, I do not see the characters.
If I re-commit each file individually, seems like the characters go away. I have done this for all the files.

**Review by ric96 — APPROVED** [review]
LGTM

**Pull request merged**